### PR TITLE
feat: display products by category on PLP

### PR DESCRIPTION
**Pull request by darocha**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/pages/Category.vue [hunk 1]
```diff
@@ -266,8 +267,17 @@ export default {
     });
 
     onSSR(async () => {
-      await search(th.getFacetsFromURL());
       await categorySearch();
+      const { categorySlug } = th.getFacetsFromURL();
+      const category = getCategoryBySlug(categorySlug, categoryTree.value?.items);
+      let producSearchParams;
```
Comment (scernisevs) [inline packages/theme/pages/Category.vue:273]: super minor but there's a typo:
produc(t)SearchParams

**Review by scernisevs — CHANGES_REQUESTED** [review]
hi @darocha ,

the PR looks good but could you please also substitute the facetGetters as part of this task?

**Review by WojtekTheWebDev — APPROVED** [review]
Fix issue reported by @scernisevs and I'll be fine for me :)

File: packages/theme/pages/Category.vue (hunk 1 above)
Comment (darocha) [reply packages/theme/pages/Category.vue]: Hi Stanis, I fixed the type and removed the facetGetters from Category.vue. Thanks!

**Review by scernisevs — APPROVED** [review]

**Pull request merged**